contract/bridge/contracts/ExchangeRateOracle.sol

```diff
@@ -60,7 +58,7 @@ contract ExchangeRateOracle is Initializable, OwnableUpgradeable {
      */
     function collateralToWrapped(uint256 amount) public view returns (uint256) {
         uint256 rate = getExchangeRate();
-        return amount.div(rate);
+        return amount.div(rate).mul(10**8).div(10**18);
```

Comment:
let's not do internal conversion to decimals. the amount pass is already in terms of 'x' ONE (1e18). similarly BTC.

Reply:
are you correct. we must do the conversion.